dll/win32/setupapi/parser.c

```diff
@@ -2327,6 +2327,10 @@ SetupDiGetINFClassW(
         ClassName, ClassNameSize, RequiredSize);
 
     /* Open .inf file */
+#ifdef __REACTOS__
```

Comment:
Is ROS ifdef actually required here? :thinking: Our setupapi is marked as forked, and the most of code is totally different.

https://github.com/reactos/reactos/blob/6b692020160ae9a925af9ae0b73f31229a107863/media/doc/WINESYNC.txt#L181

Reply:
Okay.